lib/install/validate/storage.go

```diff
@@ -32,14 +33,26 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 	defer trace.End(trace.Begin(""))
 
 	// Image Store
-	imageDSpath, ds, err := v.DatastoreHelper(ctx, input.ImageDatastoreName, "", "--image-datastore")
+	imageDSpath, ds, err := v.DatastoreHelper(ctx, input.ImageDatastorePath, "", "--image-datastore")
+
+	// provide a default path if only a DS name is provided
+	if imageDSpath.Path == "" {
+		imageDSpath.Path = path.Join(input.DisplayName, "images")
+	}
+
 	v.NoteIssue(err)
 	if ds != nil {
 		v.SetDatastore(ds, imageDSpath)
 		conf.AddImageStore(imageDSpath)
 	}
 
 	// Volume Store(s)
+	conf.BootstrapImagePath = fmt.Sprintf("[%s] %s/%s",
+		imageDSpath.Host,
+		input.DisplayName,
```

Comment:
OK so looking at this, I only moved it from `basics` to `storage` so that we have the image datastore path parsed before we do anything with the bootstrapimagepath.

If the directory already existed previously, I still don't see any kind of checks surrounding this code, so when this is created (not during validate time, note I'm just assigning these values to the VirtualContainerHostConfigSpec) if there are not checks I can add them -- in any case I don't see how moving these assignments from `basics` to `storage` would expose this issue, but I'm happy to resolve it either way.